### PR TITLE
73 add mark unmark command

### PR DESCRIPTION
**Pull request by hjungwoo01**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddTaskCommand.java [hunk 1]
```diff
@@ -61,14 +61,23 @@ private CommandResult addTask(Model model) throws CommandException {
...
 
         employee.addTask(description);
+        model.updateFilteredEmployeeList(Model.PREDICATE_SHOW_ALL_EMPLOYEES);
         System.out.println("Task added: " + description + " to employee: " + employee.getName());
         return new CommandResult(String.format(MESSAGE_SUCCESS, description));
     }
 
     @Override
-    public CommandResult execute(Model model) {
+    public CommandResult execute(Model model) throws CommandException {
```
Comment (shayaansultan) [inline src/main/java/seedu/address/logic/commands/AddTaskCommand.java:70]: Good Exception Handling

File: src/main/java/seedu/address/logic/commands/MarkTaskCommand.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    /**
+     * Getter for task number
+     * @return Task number to be marked as done
+     */
+    public int getTaskNumber() {
+        return this.taskNumber;
+    }
+}
```
Comment (shayaansultan) [inline src/main/java/seedu/address/logic/commands/MarkTaskCommand.java:77]: LGTM

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 3]
```diff
@@ -89,6 +91,12 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddTaskCommand.COMMAND_WORD:
             return new AddTaskCommandParser().parse(arguments);
 
+        case MarkTaskCommand.COMMAND_WORD:
```
Comment (shayaansultan) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:94]: Might need some more test cases

File: src/test/java/seedu/address/logic/commands/UnmarkTaskCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.commands;
```
Comment (shayaansultan) [inline src/test/java/seedu/address/logic/commands/UnmarkTaskCommandTest.java:1]: Good testing

**Review by shayaansultan — COMMENTED** [review]
LGTM!

**Review by PyromancerBoom — COMMENTED** [review]
Looks good to me!

**Pull request merged**